modules/swagger-codegen-cli/pom.xml

```diff
@@ -18,6 +18,7 @@
         <resources>
             <resource>
                 <directory>src/main/resources</directory>
+                <filtering>true</filtering>
```

Comment:
I fear there might have some side effects as we are filtering all the files under src/main/resources, not just the single one – including all the templates.

To make sure that there are none, could you actually run all the petstore samples and see if there are any changes in the output?